tests/automation/types/testing.ts

```diff
@@ -16,16 +16,19 @@ export type Group = {
 export type ConversationType = '1:1' | 'group' | 'community' | 'note-to-self';
 
 export type DMTimeOption =
-  | 'input-10-seconds'
+  | 'time-option-0-seconds'
+  | 'time-option-5-seconds'
   | 'time-option-10-seconds'
   | 'time-option-30-seconds'
-  | 'time-option-1-minute'
-  | 'time-option-5-minutes'
-  | 'time-option-1-hour'
-  | 'time-option-12-hours'
-  | 'time-option-1-day'
-  | 'time-option-1-week'
-  | 'time-option-2-weeks';
+  | 'time-option-60-seconds'
+  | 'time-option-300-seconds'
+  | 'time-option-1800-seconds'
+  | 'time-option-3600-seconds'
+  | 'time-option-21600-seconds'
+  | 'time-option-43200-seconds'
+  | 'time-option-86400-seconds'
+  | 'time-option-604800-seconds'
+  | 'time-option-1209600-seconds';
```

Comment:
because that's what desktop is using currently, the datatestids are built from the list of possible options, which are all a number of seconds (thats' also a number in seconds that we send to the recipient, etc)

Reply:
I feel like this isn't very readable, i have no idea what these numbers mean. Can we please go back to plain english versions?